tests/model_tests/test_contact.py

```diff
@@ -1,22 +1,40 @@
 import uuid
 
+#
 from core.db import db
-from core.db.entities import Contact
+from core.db.entities import Contact, Organisation
 
 
-def test_contact(app_ctx):
+def test_contact_organisation(app_ctx):
+    """
+    Test basic relationship structure b/w Contact and Organisation.
+
+    Sanity check that fk relationships are looked up as expected.
+    """
+
+    organisation = Organisation(
+        organisation_name="Evil Corp",
+        geography="Earth",
+    )
+    db.session.add(organisation)
+    db.session.flush()  # get the db-generated UUID without committing to session.
+
+    read_org = Organisation.query.first()
+    assert read_org.organisation_name == "Evil Corp"
```

Comment:
Can we call this something more generic e.g. ACME corp?

Reply:
haha Evil corp is a generic term off Mr Robot. But yeah, I can make it even more generic